src/System.Text.Json/tests/Serialization/DictionaryTests.KeyPolicy.cs

```diff
@@ -258,5 +258,55 @@ public static void KeyConflict_Serialize_WriteAll()
             // Check that we write all.
             Assert.Equal(@"{""myInt"":1,""myInt"":2}", json);
         }
+        
+        [Fact]
+        public static void CamelCaseSerialize_ApplyDictionaryKeyPolicy()
```

Comment:
Could you add equivalent tests for
- a strongly typed dictionary of POCOs: Dictionary<string, POCO>
- nested dictionary: Dictionary<string, Dictionary<string, POCO>
- class with Dictionary<string, POCO>?
- KeyValuePair<string, Dictionary<POCO>>?

Reply:
@layomia, test added.